tests/cli/test_get.py

```diff
@@ -47,35 +47,36 @@ def test_invalid_arguments(self):
 
     # MARK: All Data
 
-    @patch("scribe_data.cli.get.query_data")
-    def test_get_all_data_types_for_language(self, mock_query_data):
-        get_data(all=True, language="English")
-        mock_query_data.assert_called_once_with(
-            languages=["English"],
-            data_type=None,
-            output_dir="scribe_data_json_export",
-            overwrite=False,
-        )
-
-    @patch("scribe_data.cli.get.query_data")
-    def test_get_all_languages_for_data_type(self, mock_query_data):
-        get_data(all=True, data_type="nouns")
-        mock_query_data.assert_called_once_with(
-            languages=None,
-            data_type=["nouns"],
-            output_dir="scribe_data_json_export",
-            overwrite=False,
-        )
-
-    @patch("scribe_data.cli.get.query_data")
-    def test_get_all_languages_and_data_types(self, mock_query_data):
-        get_data(all=True, output_dir="./test_output")
-        mock_query_data.assert_called_once_with(
-            languages=None,
-            data_type=None,
-            output_dir="./test_output",
-            overwrite=False,
-        )
+    # Using sparql based data extract tests
```

Comment:
@axif0: Can you make an issue like `Outreachy Scribe-Data tests round 1` that is to write tests for all the functionality that's been added so far? The description for the issue can be as simple as:

```
In my work for Outreachy I've recently added the following functions:

- fxn_name
- another_fxn_name

This issue is to write tests for each of the above functions.
```

You can then send a PR along that adds the needed tests and then we can check it and close the issue?

Reply:
You can then write an issue like that for each issue so that you can split the work for functionality and tests up a bit? You can open a testing issue whenever you figure out some functionality that you'll be adding to the package, and we can also talk and make these issues later in Outreachy as we go through the current and new functionality to look for bugs 🐛

You'd also be welcome to open a PR and then send another commit to it to close the related testing issue :) Whatever works for you!

Reply:
@andrewtavis Sure I can make an issue to track all the merged functions I made and make tests then by sending along PR. I think it'll more convenient.